tmt/utils.py

```diff
@@ -195,6 +195,10 @@ def is_pidfile(cls, exit_code: Optional[int]) -> bool:
 DEFAULT_WAIT_TICK: float = 30.0
 DEFAULT_WAIT_TICK_INCREASE: float = 1.0
 
+# Defaults for GIT attempts and interval
+DEFAULT_GIT_CLONE_ATTEMPTS = 1
```

Comment:
the doc says 9, which one is it? as long as it is the TMT_GIT_CLONE_RETRIES env var which is not used anywhere in this patch

Reply:
Updated to be `TMT_GIT_CLONE_ATTEMPTS` and the default value is updated to be `1`.